docs/understanding-tracking-design/managing-your-data-structures/cli/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Managing data structures via the CLI"
+description: "Use the 'snowplow-cli data-structures' command to manage your data structures."
+sidebar_label: "Using the CLI"
+sidebar_position: 2
+sidebar_custom_props:
+  offerings:
+    - bdp
```

Comment:
Should cloud be included as well?

Reply:
as far as i can tell this is just how the docs differentiate between BDP and Community. There are no other options.

Reply:
Maybe I'm mixing something up, or the docs were changed since the last time I've done anything there